core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/IMCFrame.java

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Datadog, Inc. All rights reserved.
```

Comment:
I think the Datadog copyright can only added for entire new file.

Reply:
I am not sure. From what I understand from the licenses even the changes should be covered by copyright.
But IANAL - I am fine either way. 

@thegreystone - what is the 'right' way here?

Reply:
The right way is to only update the year for pre-existing files with only the Oracle copyright. For new files, the copyright of the company contributing the code can be added _after_ the Oracle copyright.

Reply:
Removed